drv/stm32h7-qspi/src/lib.rs

```diff
@@ -97,6 +97,14 @@ impl Qspi {
         self.read_impl(Command::ReadId, None, buf)
     }
 
+    /// Reads the Device unique ID buffer
+    ///
+    /// This is implemented in terms of the Winbond part; other chips may
+    /// require different commands.
```

Comment:
I kinda wonder if this ought to be called "read_winbond_unique_id` for that reason? not a big deal.

Also, since it looks like elsewhere, we ignore the first 4 bytes, it might be nice if this comment documented the meaning of the 12 bytes read into the buffer...